### PR TITLE
Update docs - part 2 (Custom timings on Tooltip)

### PR DESCRIPTION
**Pull request by benoitgrelard**
----------

- [x] Change base branch to `main` once #120 is merged. 

----------

### CODE REVIEW

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx [hunk 1]
```diff
@@ -118,6 +118,20 @@ Contains all the parts of a tooltip.
         </span>
       ),
     },
+    {
+      name: 'delayDuration',
+      type: 'number',
+      default: 700,
+      description:
+        'The duration from when the mouse enters the trigger until the tooltip gets opened.',
```
Comment (peduarte) [inline pages/primitives/docs/components/tooltip/0.0.15/index.mdx:126]: > tooltip gets opened.

that's a bit odd 🤔 

how about:
- until the tooltip opens
- until the tooltip has opened
- until the tooltip is opened

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx [hunk 2]
```diff
@@ -385,6 +399,54 @@ export default () => (
...
+      <Tooltip.Trigger>
+        <ZoomInIcon />
+      </Tooltip.Trigger>
+      <StyledContent>
+        Zoom In
+        <StyledArrow />
+      </StyledContent>
+    </Tooltip.Root>
+
+    <Tooltip.Root delayDuration={0}>
```
Comment (peduarte) [inline pages/primitives/docs/components/tooltip/0.0.15/index.mdx:437]: should this be `skipDelayDuration`?

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx (hunk 1 above)
Comment (benoitgrelard) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:126]: "until the tooltip opens" nice

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx (hunk 2 above)
Comment (benoitgrelard) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:437]: No why?

Comment (peduarte) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:437]: Oh, ok. I thought the correlation between the content and the code block was weird. 

> Use the `delayDuration` and `skipDelayDuration` to control the timings at which a tooltip is shown.

Youre talking about two different props here

> These will always show instantly:

Saying that "these" will show instantly, I thought you each button corresponds to each of the props above

then the code block shows two identical tooltips but only demoing one of the props

Comment (benoitgrelard) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:437]: Yeah I didn't want to confuse too much about showing what the other prop did, but just mentioning it as it's related to controlling "timings". Any ideas on how to make this less confusing?

Comment (peduarte) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:437]: I think one example for each, or be explicit that the example there is only about `delayDuration`

Comment (benoitgrelard) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:435]: I've ended up reducing this to just an example of the `delayDuration` prop which is by far the most common thing people want to control.

**Review by peduarte — APPROVED** [review]

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx [hunk 3]
```diff
@@ -385,6 +400,52 @@ export default () => (
 );
 ```
 
+### Show instantly
+
+Use the `delayDuration` to control the time it takes for the tooltip to open. You can define these per-instance, but it is best to set a default value for the prop so all your tooltips use the same timings.
```
Comment (jjenzz) [inline pages/primitives/docs/components/tooltip/0.0.15/index.mdx:405]: > but it is best to set a default value for the prop so all your tooltips use the same timings

I read this part and then was confused when the example doesn't show how to do this. Do you think we should show an example with `const Tooltip = extendPrimitive(Tooltip.Root, { delayDuration: 400 });`?

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx [hunk 4]
```diff
@@ -403,6 +464,10 @@ export default () => (
       keys: ['Enter'],
       description: 'If open, closes the tooltip without delay.',
     },
+    {
+      keys: ['Escape'],
+      description: 'If open, closes the tooltip without delay.',
+    },
```
Comment (jjenzz) [inline pages/primitives/docs/components/tooltip/0.0.15/index.mdx:470]: Nice catch :+1:

File: pages/primitives/docs/components/tooltip/0.0.15/index.mdx (hunk 3 above)
Comment (benoitgrelard) [reply pages/primitives/docs/components/tooltip/0.0.15/index.mdx:405]: I did think about this but wasn't sure we wanted to push that thing yet as we're not 100% sure about it right?

**Pull request merged**